virttest/utils_disk.py

```diff
@@ -793,6 +793,25 @@ def configure_empty_disk(session, did, size, ostype, start="0M", n_partitions=1,
                                       labeltype, timeout)
 
 
+def linux_disk_check(session, did):
+    """
+    Check basic functions for linux guest disk on localhost.
+    Create partition on disk, format, mount, create a file and then unmount, clean
+    partition
+
+    :param session: VM session
+    :param did: Disk Kname in VM. eg. vdb
+    """
+
+    mount_point = configure_empty_linux_disk(session, did, "100M")
```

Comment:
It looks you should call function  'clean_partition_linux' in a finally statement,  right?

code should be looks like:
`try:
    if len(mount_point) != 1:
        raise exceptions.TestError("Incorrect mount point {}".format(mount_point))
    cmd = "echo teststring >  {}/testfile".format(mount_point[0])
    if session.cmd_status(cmd):
        raise exceptions.TestError("Failed to run {}".format(cmd))
finally:
    clean_partition_linux(session, did)`

Reply:
@xutian There is a function to recover VM in test run finally block, I don't think we need to add it here.

Reply:
@Yingshun  please ensure clean code in utils level.

Reply:
@xutian  updated.

Reply:
@xutian  Can you please approve it if changes look good?